db/009-add-coreID-to-locations.sql

```diff
@@ -0,0 +1,22 @@
+DO $migration$
+    DECLARE migrationId INT;
+    DECLARE lastSuccessfulMigrationId INT;
+BEGIN
+    -- The migration ID of this file
+    migrationId := 9;
+
+    -- Get the migration ID of the last file to be successfully run
+    SELECT MAX(id) INTO lastSuccessfulMigrationId
+    FROM migrations;
+
+    -- Only execute this script if its migration ID is next after the last successful migration ID
+    IF migrationId - lastSuccessfulMigrationId = 1 THEN
+        -- ADD SCRIPT HERE
+        ALTER TABLE locations ADD COLUMN door_coreID text;
```

Comment:
We are pretty inconsistent already with our column naming scheme. However, I don't think we use capital letters anywhere right now. So may I suggest `door_coreid` and `radar_coreid`?

Could you please explain a little more about where the `core` part comes from? Is it the name or are you using it as description?

Reply:
`coreid` is a term used by particle for the UUID of their hardware devices. The radar, and door sensor Argons each have their unique coreid, hence the column names.

I will make it lowercase.

Reply:
Would it make sense then to include `argon` in the name too? Like `door_argoncoreid` and `radar_argoncoreid`. That might also stop the confusion with the `getLocationFromCoreID` (renamed to `getLocationFromArgonCoreID`) function.

Reply:
I'm not sure if it helps to have `argon `in the name, since the only `coreID` that we use pertains to Particle devices (and it encompasses all devices in their ecosystem, including photon and boron devices which we might be using in the future)